changes/ce/feat-11487.en.md

```diff
@@ -0,0 +1,3 @@
+The default number ob bcrypt rounds for pasword hashing is reduced to a more usable value by default (5).
+The number of rounds is limted to the range 5-10, because higher values consume too much CPU resources.
```

Comment:
We need a reference on why 5-10 is good enough. [This one](https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#bcrypt) - however reliable it is - recommends minimum 10.